src/com/reco1l/osu/ui/SettingsFragment.kt

```diff
@@ -0,0 +1,392 @@
+import ru.nsu.ccfit.zuev.skins.SkinManager
+import java.io.File
+
+
+enum class Section(@XmlRes val xml: Int) {
+
+    General(R.xml.settings_general),
+    Gameplay(R.xml.settings_gameplay),
+    Graphics(R.xml.settings_graphics),
+    Sounds(R.xml.settings_audio),
+    Library(R.xml.settings_library),
+    Input(R.xml.settings_input),
+    Advanced(R.xml.settings_advanced),
+
+    // Multiplayer exclusive
+    Room(R.xml.multiplayer_room_settings),
+    Player(R.xml.multiplayer_player_settings)
+
+}
+
+
+class SettingsFragment : com.edlplan.ui.fragment.SettingsFragment() {
+
+
+    private lateinit var sectionSelector: LinearLayout
+
+
+    private var section = when {
+
+        Multiplayer.isRoomHost -> Section.Room
+        Multiplayer.isMultiplayer -> Section.Player
+
+        else -> Section.General
+    }
+
+
+    override fun onLoadView() {
+
+        sectionSelector = findViewById(R.id.section_selector)!!
+        sectionSelector.removeAllViews()
+
+        fun createSectionButton(text: String, icon: Int, section: Section) {
+
+            val button = TextView(ContextThemeWrapper(context, R.style.settings_tab_text))
+
+            button.layoutWidth = 200f.dp
+            button.text = text
+            button.drawableLeft = requireContext().getDrawable(icon)!!
+            button.drawableLeft!!.setTint(Color.WHITE)
+
+            button.setOnClickListener {
+
+                sectionSelector.forEach {
+                    if (it == button) {
+                        it.setBackgroundResource(R.drawable.rounded_rect)
+                        it.background.setTint(0xFF363653.toInt())
+                    } else {
+                        it.background = null
+                    }
+                }
+
+                this.section = section
+                setPreferencesFromResource(section.xml, null)
+            }
+
+            sectionSelector.addView(button)
+
+            if (this.section == section) {
+                button.callOnClick()
+            }
+        }
+
+
+        if (Multiplayer.isMultiplayer) {
+
+            fun createDivider(text: String) {
+
+                val divider = TextView(ContextThemeWrapper(context, R.style.settings_tab_divider))
+                divider.text = text
+
+                sectionSelector.addView(divider)
+            }
+
+            createDivider("Multiplayer")
+            createSectionButton("Player", R.drawable.person_24px, Section.Player)
+            createSectionButton("Room", R.drawable.groups_24px, Section.Room)
+            createDivider("Game")
+
+        }
+
+
+        if (!Multiplayer.isMultiplayer) {
+            createSectionButton("General", R.drawable.grid_view_24px, Section.General)
+        }
+
+        createSectionButton("Gameplay", R.drawable.videogame_asset_24px, Section.Gameplay)
+        createSectionButton("Graphics", R.drawable.display_settings_24px, Section.Graphics)
+        createSectionButton("Sound", R.drawable.headphones_24px, Section.Sounds)
```

Comment:
```suggestion
        createSectionButton("Audio", R.drawable.headphones_24px, Section.Sounds)
```

Same here. I think `Audio` is better.